src/libs/ReportUtils.js

```diff
@@ -1494,6 +1494,9 @@ function getProperSchemaForModifiedExpenseMessage(newValue, oldValue, valueName,
 /**
  * Get the report action message when expense has been modified.
  *
+ * ModifiedExpense::getNewDotComment in Web-Expensify should match this.
+ * If we change this function be sure to update the backend as well.
+ *
```

Comment:
What's the reasoning for this? Why do we need to match in both?

Reply:
The frontend uses this logic to format the message text for modified expense actions. I added the same logic [here](https://github.com/Expensify/Web-Expensify/blob/340691674c12d3cf5ff64545a64bbc126f05a273/lib/Report/Action/ModifiedExpense.php#L204) in Web-E so we can have the same format for iOS and Android push notifications